docs: SDK and CLI book findings + fixes (#2658)
openvm-org/openvm: docs/vocs/docs/pages/book/getting-started/quickstart.mdx

cargo openvm keygen
```

> This will output a serialized proving key `app.pk`, a verification key `app.vk`, and an aggregation prefix proving key `agg_prefix.pk` into `${target-dir}/../openvm/` (by default `./openvm/`). Use `--app-only` to skip generating the aggregation key, which is used in STARK and EVM proving.

Reviewer: ```suggestion
This will output a serialized proving key `app.pk`, a verification key `app.vk`, and an aggregation prefix proving key `agg_prefix.pk` into `${target-dir}/../openvm/` (by default `./openvm/`). Use `--app-only` to skip generating the aggregation key, which is only needed for STARK and EVM proving.
```
